pcf_test.go

```diff
@@ -281,3 +270,37 @@ func TestUpdatePcfSubsriberPolicyDataDel(t *testing.T) {
 		t.Errorf("test case failed\n")
 	}
 }
+
+func TestGetBitRateUnit(t *testing.T) {
+	fmt.Printf("test case TestGetBitRateUnit \n")
+	val, unit := service.GetBitRateUnit(1000)
+	if val != 1 || unit != " Kbps" {
+		t.Errorf("test case failed. Expected: 1 Kbps, Received: %v%v\n", val, unit)
+	}
+	val, unit = service.GetBitRateUnit(67200)
```

Comment:
Please use map of input value vs expected outcome and iterate over that map. that shall reduce overall code.